data/hai/hai outfits.txt

```diff
@@ -11,6 +11,38 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+outfit "Phantom Pallet"
+	category "Systems"
+	cost 12000
+	thumbnail "outfit/phantom pallet"
+	"mass" 0.2
+	"unplunderable" 1
+	"energy consumption" 0.016667
+	"heat generation" 0.03334
+
+	# Replacing 30% of your cargo bay with these triples your cargo scan time.
+	"cargo space" -1
+	"cargo scan opacity" 15
+
+	description "This borderline-illegal smuggling tool is an Unfettered modification of the standard Hai Holopad. It is durable enough to survive a firefight and is hard to detect."
+	description "Holopads are small versions of the Hai Holovid Zone. They have holographic projectors, textured force fields, and phase array speakers. As a mid-priced but powerful option, it's often seen in meeting rooms, arcades, and personal gaming setups. Several can be packed together to make a larger holographic space."
+	description "Unfettered Phantom Pallets can hide under floors, in walls, or in ceilings while remaining undetected. Install them in your cargo bay and project fake cargo as sensor ghosts. This confuses scanners into scanning empty space instead of contraband. That can make the difference between seized cargo and delivering needed supplies."
```

Comment:
```suggestion
	description "This borderline-illegal smuggling tool is an Unfettered modification of standard Hai holographic technology, durable enough to survive a firefight and easily hidden under floors, in walls, or in ceilings while remaining undetected. An enterprising privateer might pad their cargo bay with these pallets and project fake cargo as sensor ghosts, confusing scanners into scanning empty space instead of contraband; that extra time has been known to make the difference between seized cargo and delivering needed supplies."
```

Reply:
This changes the meaning of the outfit from:

- Unfettered made a simple modification to an outfit that is basically identical

To

- This is a clever new Unfettered device they made with holographic technology

The reason I went with the former is because someone earlier in the comment chain didn't want the Unfettered to waste resources making novel holographic outfits. Having it be a simple modification of a Holopad means the Unfettered were using their resources effectively.

Also, you're removing some interesting color, about the Holopad's use in Hai arcades, and the like.

Reply:
I didn't think repeating the description about arcades was necessary when it was already covered in the holovid zone's description. I'll try to adjust it.

Reply:
They are separate outfits from separate factions. Players may encounter them in either order.

Reply:
Yes, but that's not really its main purpose, so I thought it was redundant.

Reply:
```suggestion
	description `Hai holographic outfits are made up of a set of individual "holopads" which work together to create a larger scene. The Phantom Pallet is an Unfettered variation of these holopads, durable enough to survive a firefight and easily hidden under floors, in walls, or in ceilings while remaining undetected. An enterprising privateer might pad their cargo bay with these and project fake cargo as sensor ghosts, confusing scanners into scanning empty space instead of contraband; that extra time has been known to make the difference between seized cargo and delivering needed supplies.`
```
How about this?